cmd/gossamer/README.md

```diff
@@ -0,0 +1,127 @@
+  details of how Gossamer orchestrates a blockchain client are [described below](#client-components).
+  - `chain` - specifies the [chain configuration](../../chain) that the
+    [Gossamer host node](https://chainsafe.github.io/gossamer/getting-started/overview/host-architecture/) should load
+  - `log` - supports levels `crit` (silent) to `trce` (detailed), default is `info`
+  - `name` - node name, as it will appear in, e.g., [telemetry](https://telemetry.polkadot.io/)
+- `account` - The `account` subcommand provides the user with capabilities related to generating and using `ed25519`,
+  `secp256k1`, and `sr25519` [account keys](https://wiki.polkadot.network/docs/learn-keys), and managing the keys
+  present in the [Gossamer keystore](#keystore). The action for this subcommand is defined in [account.go](account.go);
+  it is an interface to the capabilities defined in the [`crypto`](../../lib/crypto) and
+  [`keystore`](../../lib/keystore) packages. This subcommand provides capabilities that are similar to
+  [Parity's Subkey utility](https://docs.substrate.io/v3/tools/subkey).
+  - `generate` - creates a new key pair; specify `--ed25519`, `secp256k1`, or `--sr25519` (default)
+  - `list` - lists the keys in the Gossamer keystore
+  - `password` - allows the user to provide a password for encrypting a generated key, or unlocking the Gossamer
+    keystore
+- `import-runtime` - This subcommand takes a [Wasm runtime binary](https://wiki.polkadot.network/docs/learn-wasm) and
+  uses it to generate a [genesis](https://wiki.polkadot.network/docs/glossary#genesis) configuration file; it does not
+  require any flags, but expects the path to a Wasm file to be provided as a command-line parameter (example:
+  `./bin/gossamer import-runtime runtime.wasm > genesis.json`).
+- `build-spec` - This subcommand allows the user to "compile" a human-readable Gossamer genesis configuration file into
+  a format that the Gossamer node can consume. If the `--genesis` parameter is not provided, the generated genesis
+  configuration will represent the Gossamer default configuration.
+  - `genesis` - path to the human-readable configuration file that should be compiled into a format that Gossamer can
+    consume
+- `import-state` - The `import-state` subcommand allows a user to seed [Gossamer storage](../../dot/state) with
+  key-value pairs in the form of a JSON file. The input for this subcommand can be retrieved from
+  [the state_getPairs RPC endpoint](https://github.com/w3f/PSPs/blob/master/PSPs/drafts/psp-6.md#1114-state_getpairs).
+  - `first-slot` - the first [BABE](https://wiki.polkadot.network/docs/learn-consensus#block-production-babe) slot,
+    which is the timestamp of the first block divided by the slot duration
+  - `header` - path to a JSON file that describes the block header corresponding to the given state
+  - `state` - path to a JSON file that contains the key-value pairs with which to seed Gossamer storage
```

Comment:
To me state is more of a runtime thing, whilst storage is more of a disk (and runtime) storage. Although you're right, in this case, they both work I think? Anyway I think it would be good to use only one of the two, leaning towards `state`. You can add that the `state` is stored here and there too.

Reply:
I agree that "state" should imply runtime storage, but I don't know if that is the case for the Gossamer code base as a whole so I don't feel like it's necessary to worry about that in this document at this time.